src/main/kotlin/net/ccbluex/liquidbounce/features/module/modules/world/scaffold/ModuleScaffold.kt

```diff
@@ -540,6 +552,15 @@ object ModuleScaffold : Module("Scaffold", Category.WORLD) {
                     }
                 }
 
+                SameYMode.HYPIXEL -> {
+                    if (player.velocity.y == -0.15233518685055708 && jumps >= 2) {
+                        jumps = 0
+                        return BlockPos(blockPos.x, startY, blockPos.z)
+                    } else {
+                        return BlockPos(blockPos.x, startY - 1, blockPos.z)
+                    }
+                }
+
                 else -> {}
             }
```

Comment:
**Here is an approach (This may well be not the best one) to 'bypass' detekt in this case:** 

The entire 'not towering' logic could be extracted into a function.
For example:

```
private fun getNotToweringTargetedPosition(blockPos: BlockPos) : BlockPos? {
        when(sameYMode) {
            SameYMode.ON -> {
                return BlockPos(blockPos.x, placementY, blockPos.z)
            }

            SameYMode.FALLING -> {
                if (player.velocity.y < 0.2) {
                    return BlockPos(blockPos.x, placementY, blockPos.z)
                }
            }

            SameYMode.HYPIXEL -> {
                var offset = 1
                if (player.velocity.y == -0.15233518685055708 && jumps >= 2) {
                    jumps = 0
                    offset = 0
                }
                return BlockPos(blockPos.x, startY - offset, blockPos.z)
            }

            else -> {}
        }

        return null
    }
```

Here is what's supposed to be in the getTargetedPosition() function instead:
```
...
        if (!isTowering) {
            val result = getNotToweringTargetedPosition(blockPos)
            if (result != null) {
                return result
            }
        }
...
```

Reply:
I just saw your response after I already completed my own refactor, but it's basically the same approach as yours:
https://github.com/CCBlueX/LiquidBounce/pull/3799/commits/9b5e608fa3e87c249e6d4364a291b54c850ca170